src/main/target/FLYWOOF411/target.c

```diff
@@ -40,5 +40,5 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM3, CH4, PB1,  TIM_USE_ANY,   0, 0), //  RSSI   1,2
     DEF_TIM(TIM5, CH4, PA3,  TIM_USE_ANY,   0, 1), //  RX2    1,0
     DEF_TIM(TIM2, CH3, PB10, TIM_USE_CAMERA_CONTROL,   0, 0), //  CAM CC 1,1
-    DEF_TIM(TIM2, CH4, PB11, TIM_USE_LED,   0, 0), //  LED    1,6
+    DEF_TIM(TIM2, CH1, PA15, TIM_USE_LED,   0, 0), //  LED    1,5
```

Comment:
Why is this being changed? If any boards with the design using PB11 as the LED_STRIP pin were sold to customers this will break the LED_STRIP functionality for them.

Reply:
We haven't sold any boards yet, so don't worry. Since the F411 chip doesn't have PB11, it needs to be modified to PA15, which is the previous mistake. thanks you help!